test/IISExpress.FunctionalTests/InProcess/AppOfflineTests.cs

```diff
@@ -5,11 +5,13 @@
 using System.IO;
 using System.Net;
 using System.Net.Http;
+using System.Threading;
```

Comment:
I don't believe these tests will be sufficient. I think we need something for app_offline with full IIS for this PR. We may also want to consider adding IIS specific shutdown tests as that can be triggered via configuration changes or global recycle.

Reply:
Moved and enabled ones the work. Filed https://github.com/aspnet/IISIntegration/issues/1074 in addition to existing https://github.com/aspnet/IISIntegration/issues/1026